app/scripts/modules/core/src/search/global/GlobalSearchResults.tsx

```diff
@@ -0,0 +1,79 @@
+import * as React from 'react';
+import { UISref } from '@uirouter/react';
+
+import { SETTINGS } from 'core/config/settings';
+import { ISearchResult } from 'core/search/search.service';
+import { ISearchResultSet } from 'core/search/infrastructure/infrastructureSearch.service';
+import { SearchResult } from 'core/search/infrastructure/SearchResult';
+
+export interface IGlobalSearchResultsProps {
+  categories: ISearchResultSet[];
+  query: string;
+  onItemKeyDown: (event: React.KeyboardEvent<HTMLAnchorElement>) => any;
+  onResultClick: (result: ISearchResult) => any;
+  onSeeMoreClick: Function;
+  resultRef: (categoryIndex: number, resultIndex: number, ref: HTMLAnchorElement) => any;
+  seeMoreRef: (ref: HTMLAnchorElement) => any;
+}
+
+export function GlobalSearchResults({
+  categories,
+  query,
+  onItemKeyDown,
+  onResultClick,
+  onSeeMoreClick,
+  resultRef,
+  seeMoreRef
+}: IGlobalSearchResultsProps) {
+  const { searchVersion } = SETTINGS;
+
+  if (!categories.length) {
+    return (
+      <ul className="dropdown-menu" role="menu">
+        <li className="result">
+          <a>No matches</a>
```

Comment:
nitpick: If this isnt really a link, we can make this a normal <div>/<span> and style it to look like a link.

Reply:
👍 i don't really know why, but for some reason i inadvertently put DOM structure/CSS mods out of scope for this PR, even though that's likely nonsensical.

Reply:
Just to get down what we talked about in person into writing: turns out these styles rely on a big chain of overridden bootstrap stuff that is specific to `a` tags inside `li`s, and trying to modify it is a big rabbit hole. For now I'm gonna leave these as-is and hopefully we'll make some time to make this group of styles better across the codebase rather than doing a bunch of one-off hacks.